spec/factories/posts.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :post do
+    caption { "MyString" }
+    status { 1 }
+    category { 1 }
+    latitude { "9.99" }
+    longitude { "9.99" }
```

Comment:
1. `latitude` and `longitude` are decimals in the database if i remember. should this mean you can put numbers in lines 6 + 7 instead of strings?
2. don't we need an association with the images here also?

Reply:
Yes, good catch! We changed the coordinates to floats (with decimals) since that is the format we will send in the post request later.

Regarding image association, we think we don't need that since with Active Storage the image is 'attached' to the Post, without actually being its own database column:

<img width="571" alt="Screenshot 2019-06-13 at 13 02 56" src="https://user-images.githubusercontent.com/48205436/59427729-a1000c80-8ddb-11e9-96c2-0da03ed8b00b.png">